src/object/icecrusher.cpp

```diff
@@ -179,6 +221,52 @@ IceCrusher::collision_solid(const CollisionHit& hit)
         set_state(RECOVERING);
       }
       break;
+	case CRUSHING_RIGHT:
```

Comment:
Consistent line indentation and brace style...braces in a new line. I hate Java style braces, and I hate Java.

Reply:
Fixed... I presume?  The other cases don't do this but I did it anyway.